blue_hardware_interface/src/blue_kinematics.cpp

```diff
@@ -28,20 +28,25 @@ void BlueKinematics::init(
   transmissions_.resize(num_transmissions_);
   actuator_states_.resize(num_transmissions_);
   actuator_commands_.resize(num_transmissions_);
+  actuator_position_commands_.resize(num_transmissions_);
   joint_states_.resize(num_transmissions_);
   joint_commands_.resize(num_transmissions_);
+  joint_position_commands_.resize(num_transmissions_);
 
   actuator_pos_.resize(num_joints_);
   actuator_vel_.resize(num_joints_);
   actuator_eff_.resize(num_joints_);
-  actuator_cmd_.resize(num_joints_);
+  actuator_eff_cmd_.resize(num_joints_);
+  actuator_pos_cmd_.resize(num_joints_);
   joint_pos_.resize(num_joints_);
   joint_vel_.resize(num_joints_);
   joint_eff_.resize(num_joints_);
-  joint_cmd_.resize(num_joints_);
+  joint_eff_cmd_.resize(num_joints_);
+  joint_pos_cmd_.resize(num_joints_);
 
   joint_offsets_.resize(num_joints_, 0.0);
-  raw_joint_cmd_.resize(num_joints_, 0.0);
+  raw_joint_eff_cmd_.resize(num_joints_, 0.0);
+  raw_joint_pos_cmd_.resize(num_joints_, 0.0);
```

Comment:
this is 100% my fault but the variable naming here is super confusing; let's fix in a future PR

Reply:
sounds good!